### PR TITLE
Add the search template

### LINKED ISSUE #9: Update search template
Update the search template so that it matches the [design](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020078%3A3812). Example PRs [here](https://github.com/Automattic/themes/pull/6518) and [here](https://github.com/Automattic/themes/pull/6599).

## Desktop
Screen Shot 2022-09-29 at 6 12 23 PM

## Mobile
Note that we don't have mobile designs for this template; however, we can do something similar to the [single post template](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020126%3A4278), where the sidebar content is displayed below the search page content on mobile.

#### Scenario: Search template
**Given** The Sensei theme is activated
**When** I view the search page (e.g. http://one.wordpress.test/?s)
**Then** the page looks similar to the design.

### PR DESCRIPTION
**Pull request by gabrielcaires**
### Note:
* I checked if the CSS additions could be done via theme.json, I only left the necessary ones.
* I am going to implement the pagination on ticket [Update archive template#10](https://github.com/Automattic/sensei-theme/issues/10)

### Changes
* Update the search template to match the layout

## Testing instructions
Access 
http://[YOUR_LOCAL_ENV]/?s=

Check if it matches the layout.
Screen Shot 2022-10-12 at 20 11 36

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -156,3 +156,39 @@ a {
...
+/* Increase the default vertical distance between elements when inside a layout flow */
+body .is-layout-flow .wp-block-group {
+   margin-block-end: 5rem;
+}
+
+/*
+ * Search styles
+ */
+.wp-block[data-type="core/search"] {
+	margin-bottom: 5rem !important;
```
Comment (LavaGolem) [inline style.css]: can we remove the **!important** here and handle the margin-bottom  in different way?

File: templates/search.html [hunk 2]
```diff
@@ -1,31 +1,27 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group"><!-- wp:search /-->
+<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"var:preset|spacing|70","left":"var:preset|spacing|70"}}}} -->
+<div class="wp-block-columns"><!-- wp:column {"width":"72%","layout":{"type":"constrained"}} -->
+<div class="wp-block-column" style="flex-basis:72%"><!-- wp:heading {"level":1} -->
+<h1>SEARCH</h1>
```
Comment (LavaGolem) [inline templates/search.html]: We should translate all the text

Comment (gabrielcaires) [reply templates/search.html]: Fixed [be5e087](https://github.com/Automattic/sensei-theme/pull/36/commits/be5e087adc08d44d0936a995c65d453f94896430) and  [432f8b6](https://github.com/Automattic/sensei-theme/pull/36/commits/432f8b6047b1a9ae517938295e59b0ffb2026090)

Comment (gabrielcaires) [reply templates/search.html]: Fixed here   [be5e087](https://github.com/Automattic/sensei-theme/pull/36/commits/be5e087adc08d44d0936a995c65d453f94896430) and [432f8b6](https://github.com/Automattic/sensei-theme/pull/36/commits/432f8b6047b1a9ae517938295e59b0ffb2026090)

File: style.css (hunk 1 above)
Comment (gabrielcaires) [reply style.css]: Fixed here [91c65d7](https://github.com/Automattic/sensei-theme/pull/36/commits/91c65d760673ce5c07ca5034d273ed85f4283c34)

**Review by LavaGolem — APPROVED** [review]

**Pull request merged**